client/src/utils.ts

```diff
@@ -161,3 +167,30 @@ export const deepHtmlStringMatch = (element: any, match: string) => {
     .map((child: any) => deepHtmlStringMatch(child, match))
     .reduce((accum: boolean, cur: boolean) => accum || cur);
 };
+
+// Given a HTML img element, process image data for barcode string
+export const processImageBarcode = async (img: HTMLImageElement) => {
+  const codeReader = new BrowserMultiFormatReader();
+  if (img) {
+    const result = await codeReader
+      .decodeFromImage(img)
+      .then((res: any) => res.text)
+      .catch((err: any) => null);
+    if (result) {
+      return result;
+    }
+  }
+  return false;
+};
+
+// Jest does not load up image files by default
+// in order to actually read file data, we use nodejs "fs" module
+export const jestImportImage = (imgPath: string) => {
```

Comment:
where is this used?

Reply:
This will be used in #166 when we can get the library to run in jest testing env.